README.md

```diff
@@ -84,6 +84,11 @@ set AR_x86_64_unknown_none=llvm-ar
 
 Please follow [Secure Boot Guide](doc/secure_boot_guide.md)
 
+### Before build
+```
+git submodule update --init --recursive
+sh sh_script/preparation.sh
```

Comment:
It would be safer to specify the actual shell specified in this script:

```suggestion
bash sh_script/preparation.sh
```

Alternatively, this PR could just change the perms on the script to allow the user to run:

```suggestion
./sh_script/preparation.sh
```